code/modules/antagonists/changeling/powers/apex_predator.dm

```diff
@@ -0,0 +1,26 @@
+/datum/action/changeling/apex_predator
+	name = "Apex Predator"
+	desc = "We evolve a keen intuition, allowing us to detect the anxieties of nearby lifeforms."
```

Comment:
This is always gonna point at every coder :(

Reply:
:(